docs/tutorials/otio-serialized-schema.md

```diff
@@ -361,21 +361,21 @@ parameters:
 *documentation*:
 
 ```
-None
+Refers to a sequence of image files to be played back at a constant rate.
 ```
 
 parameters:
 - *available_range*: 
-- *frame_step*: 
-- *frame_zero_padding*: 
+- *frame_step*: Step between frame numbers in the image sequence.
+- *frame_zero_padding*: Number of digits to pad zeros out to in the image names.
```

Comment:
Could this be simplified to something like: Number of digits in frame number

Reply:
How about: "Number of digits in image name frame number."?
Just want to make sure I'm explicit about this being for how the file name is generated. Though I could also clarify that more in the docstring for the class.

Reply:
Coming from studios that don't pad our frame numbers, I worry a bit about simply "number of digits...". For example frame "foo.100.exr" should not be a pad of "3" for us since we'll want to see "99" and not "099" for the previous frame. The argument is pretty descriptive though but I would just ask that the doc string continue to keep "pad" somehow if possible.

Reply:
`"Number of digits in image name frame number."`
is fine by me :)

Reply:
@avrata do you think having the word "padding" in the attribute name is good enough, or do you feel like the behavior should be outlined more explicitly in the docstring?

Thank you for the help hashing this out!

Reply:
Yep, it's probably fine. I was going to comment on the danger of: https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/633/commits/bb2bd99c3ba2825df8d672a079f9d35ad5035aad#diff-c166f6447c3991e9810480f19426caaaR255 but saw that got fixed that up before I could mention it. The attribute is descriptive enough.

Reply:
This whole no padding point. Have we covered that in the schema? I can't remember the default value of `frame_zero_padding`. If it's set to 1 I guess we're ok?

Reply:
I think so, the default is "0" https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/602/files#diff-06c32a0f949112099a7b59156703bba9R29 which works great for me :) (from non-padding studios)

Reply:
Ok, great! Was worried that some of the other methods would freak out, but @reinecke knows his stuff :)